packages/server-ct/example/webpack.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  output: {
+    publicPath: '/',
+  },
+  devServer: {
+    publicPath: '/',
+  },
+}
```

Comment:
We should make these the default values in our base webpack.config.js inside of @cypress/webpack-devserver -- I'll add a story